tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -818,7 +818,8 @@ def test_vpc_config(
     def test_invalid_invoke(self, aws_client, snapshot):
         with pytest.raises(aws_client.lambda_.exceptions.ClientError) as e:
             aws_client.lambda_.invoke(
-                FunctionName="arn:aws:lambda:us-east-1:123400000000@function:myfn", Payload=b"{}"
+                FunctionName=f"arn:aws:lambda:{aws_client.lambda_.meta.region_name}:123400000000@function:myfn",
```

Comment:
nit: we can use the `region_name` fixture here.

Reply:
`FunctionName=f"arn:aws:lambda:{region_name}:{account_id}@function:myfn",`

Reply:
The account id should not be an actually used one, it's just some random number, so I do not need the fixture here. For the region, we use this pattern all over lambda tests, so I did not want to change that yet :)

Reply:
Aah got it, although we can case once in a trillion where the `account_id` could be `123400000000` but that should be fine.

Reply:
That should be fine but we lately have a bunch of places in lambda where we use the fixtures: https://github.com/localstack/localstack/blob/3b8099b864d7cdf684fc630a2c6d09f8d3abab35/tests/aws/services/lambda_/test_lambda_api.py#L424

Reply:
Ah yes, you are absolutely right! We should definitely bring all that into line at some point. If you do not have a strong opinion on this (which I would be fine with as well) I would leave it as is for now, and migrate all of the remaining tests to the new fixture for easier readability at a later point :)

Reply:
Yes that should be fine for now! 🙌